spaceMissionManager-service/src/test/java/cz/muni/fi/services/UserServiceTest.java

```diff
@@ -0,0 +1,354 @@
+package cz.muni.fi.services;
+
+import cz.muni.fi.config.ServiceConfiguration;
+import cz.muni.fi.dao.MissionDao;
+import cz.muni.fi.dao.UserDao;
+import cz.muni.fi.entity.Mission;
+import cz.muni.fi.entity.User;
+import cz.muni.fi.services.UserService;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.DataAccessException;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.testng.AbstractTestNGSpringContextTests;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import sun.reflect.generics.reflectiveObjects.NotImplementedException;
+
+import java.util.*;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.*;
+
+
+/**
+ *
+ * @author Vojtech Bruza
+ */
+
+@ContextConfiguration(classes = ServiceConfiguration.class)
+public class UserServiceTest extends AbstractTestNGSpringContextTests {
+    @Mock
+    private UserDao userDao;
+    @Mock
+    private MissionDao missionDao;
+
+    @Autowired
+    @InjectMocks
+    private UserService userService;
+
+    private Long idCounter;
+    private Map<Long, User> users = new HashMap<>();
+    private Map<Long, Mission> missions = new HashMap<>();
```

Comment:
You never use this

EDIT: My bad, i get it now, this is fine

Reply:
Can I ask what did you think is wrong?